content/github/getting-started-with-github/access-permissions-on-github.md

```diff
@@ -17,16 +18,42 @@ A repository owned by a user account has two permission levels: the *repository
 
 ### Organization accounts
 
+#### Code Block Component
+<!--react-->
+<CodeBlock language="javascript"
```

Comment:
Looks like the React component adds languages, so we should be able to drop this in the Markdown?
```suggestion
<CodeBlock
```